git.html.markdown

```diff
@@ -238,7 +241,7 @@ $ git clone -b master-cn https://github.com/adambard/learnxinyminutes-docs.git -
 ### commit
 
 Stores the current contents of the index in a new "commit." This commit contains
-the changes made and a message created by the user.
+the changes made and a message created by the user. *If you do not add the "-m" flag, git will throw you in to Vim or Nano, and your world will explode.*
```

Comment:
It did, did u want me to merge in to master on my local or were you going
to merge my branch in to master?

On Thu, Oct 15, 2015 at 2:35 PM, Levi Bostian <notifications@github.com>
wrote:

> In git.html.markdown
> <https://github.com/adambard/learnxinyminutes-docs/pull/1487#discussion_r42162522>
> :
>
> > @@ -238,7 +241,7 @@ $ git clone -b master-cn https://github.com/adambard/learnxinyminutes-docs.git -
> >  ### commit
> >
> >  Stores the current contents of the index in a new "commit." This commit contains
> > -the changes made and a message created by the user.
> > +the changes made and a message created by the user. *If you do not add the "-m" flag, git will throw you in to Vim or Nano, and your world will explode.*
>
> Haha is that right? I am pretty sure that happened to me the first time
> that happened to me too.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/adambard/learnxinyminutes-docs/pull/1487/files#r42162522>
> .
>